pkg/api/v1/conversion.go

```diff
@@ -36,7 +36,7 @@ func addConversionFuncs() {
 	}
 
 	// Add field conversion funcs.
-	err = api.Scheme.AddFieldLabelConversionFunc("v1", "Pod",
+	err = api.Scheme.AddFieldLabelConversionFunc(api.Group, "v1", "Pod",
```

Comment:
Will it be better to pass TypeMeta here instead of the individual params?
That way, we wont have to change the func signatures when we add support for provider later, or merge/unmerge group and version together.